uast/transformer/semantic.go

```diff
@@ -147,25 +147,36 @@ func (c *commentElems) isTab(r rune) bool {
 }
 
 func (c *commentElems) Split(text string) bool {
-	if !strings.HasPrefix(text, c.StartToken) || !strings.HasSuffix(text, c.EndToken) {
+	trimmed := strings.TrimSpace(text)
```

Comment:
This shouldn't happen automatically. It belongs to a separate op, or should be specified explicitly as a flag.

Reply:
Sorry, I rebased and this got fakely outdated. You mean like for example adding a new "trimText" field to `commentUAST` and a new op like `CommentTextTrimmed`?

Reply:
Yes, something like this will be nice :)

But if positions are off the node will fail the token verification for sure. So fixing positions should fix both issues (validation and trim).

Reply:
Done. In the case of Python it works because it only happens with comments on their own line that are (sometimes) indented. In the native AST those doesn't have position, unfortunately, trough in this case it should be probably easy to fix.